src/index.html

```diff
@@ -6,9 +6,44 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="index.html" class="nav__logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="navBox">
+        <ul class="liBox">
+          <li>
+            <a  href="index.html" class="is-active"><span class="is-active">Apple</span></a>
+          </li>
+          <li>
+            <a  href="index.html"><span>Samsung</span></a>
```

Comment:
Span is redundant here
```suggestion
            <a href="index.html">Samsung</a>
```

Reply:
Hello.
Without <span> it is not possible to correctly perform positioning ::after, since it shifts the tag <a>.

Reply:
consider positioning `::after` absolutely in relation to links.
If you have problems with it, please write in the chat